app/models/podcast_episode.rb

```diff
@@ -39,6 +41,14 @@ class PodcastEpisode < ApplicationRecord
   after_commit :index_to_elasticsearch, on: %i[update]
   after_commit :remove_from_elasticsearch, on: [:destroy]
 
+  # [@atsmith813] this is adapted from the `search_field` property in
+  # `config/elasticsearch/mappings/feed_content.json` and
+  # `app/serializers/search/podcast_episode_serializer.rb` with a couple of
+  # extras
+  pg_search_scope :search_podcast_episodes,
+                  against: %i[body subtitle title],
+                  using: { tsearch: { prefix: true } }
+
```

Comment:
Currently, with Elasticsearch we're searching against the result of the model method `body_text`:
https://github.com/forem/forem/blob/8a7179167c5c04ac03dd6333168c0b4836a46e8d/app/models/podcast_episode.rb#L72-L74

We can still mimic that exact behavior by saving this same result as its own column. I'm open to that or other suggestions. This PR right now, instead, searches directly against `body`, `title`, and `subtitle`. `body` does still contain HTML tags and such, but I don't think it will interfere too much with results, especially since we added `title` and `subtitle`. We can always adjust this in the future too.

Reply:
It shouldn't matter: 

```sql
Forem_development=# select to_tsvector('simple', '<tag>text</tag>');
 to_tsvector
-------------
 'text':1
(1 row)
```

the FTS query won't see the HTML

Reply:
Oh...right 😅  something, something...are we there yet? 🤣 🙈